src/Authoring/WinRT.SourceGenerator/Generator.cs

```diff
@@ -1,258 +1,342 @@
-﻿using Microsoft.CodeAnalysis;
```

Comment:
Note: I normalized whitespaces in this file, review with whitespace diff off.
The actual changes are just a few lines of code being added.